index.js

```diff
@@ -79,6 +79,58 @@
 
   var _ = R.__;
 
+  //  placeholder :: a -> Boolean
+  var placeholder = function(x) {
+    return x != null && x['@@functional/placeholder'] === true;
+  };
+
+  /* istanbul ignore next */
+  var Any = function Any() {};
+
+  var arity = function(n, f) {
+    switch (n) {
+      case 0: return function() { return f.apply(this, arguments); };
+      case 1: return function(a) { return f.apply(this, arguments); };
+      case 2: return function(a, b) { return f.apply(this, arguments); };
+      case 3: return function(a, b, c) { return f.apply(this, arguments); };
+    }
+  };
```

Comment:
`R.arity` was deprecated in v0.15.0, so we must provide this ourselves. It's not onerous since we don't export any functions with arity greater than three.